charts/rancher-monitoring/v0.1.3/charts/operator/templates/deployment.yaml

```diff
@@ -48,8 +49,6 @@ spec:
             - --prometheus-config-reloader={{ template "system_default_registry" . }}{{ .Values.image.prometheusConfigReloader.repository }}:{{ .Values.image.prometheusConfigReloader.tag }}
             - --config-reloader-image={{ template "system_default_registry" . }}{{ .Values.image.configmapReload.repository }}:{{ .Values.image.configmapReload.tag }}
             - --labels={{ .Values.apiGroup }}=true
-            - --manage-crds={{ .Values.manageCRDs }}
-            - --with-validation={{ .Values.withValidation }}
```

Comment:
Part of v0.39.0 changes removed these fields: https://github.com/prometheus-operator/prometheus-operator/blame/99ecd7100c2dd9c1392e7670215045cf41606a0f/CHANGELOG.md#L68

Reply:
Yes, you can use `rancher_min_version: <2.x.x version>` where version is the minimum version of rancher that supports v1.16+. You can use the following to figure that out:
https://rancher.com/support-maintenance-terms/all-supported-versions/rancher-v2.3.6/

Example questions.yaml:
https://github.com/rancher/charts/blob/6aaa35cacac454348ef9dce75f8e8c2508137663/charts/longhorn/v0.4.1/questions.yml#L1

Reply:
oops replied to the wrong one, but you got it anyways 😁

Reply:
Hmm so currently this minimum version is set to: https://github.com/rancher/system-charts/blob/dc3df5b5342b8210861bf800420589117feafc89/charts/rancher-monitoring/v0.1.3/questions.yml#L1.

But this change needs to be backported to be supported in either/both `2.3.8+` or `2.4.6+`, both of which currently support k8s 1.15 based on the doc that you sent.

@deniseschannon any thoughts here?